### PR TITLE
Edit: CSS overflow

### LINKED ISSUE #4415: [Edit] CSS [Edit] CSS overflow: overflow
### Type of Edit (select all that apply)

Add new content (definitions, codeblocks, etc.)

### Description (optional)

We would like an edit to a term within CSS `overflow`. The term entry lives in `content/css/concepts/overflow/terms/overflow/overflow.md`

Please refer to the [term entry templates](https://github.com/Codecademy/docs/blob/main/documentation/term-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

Please:
1) Add supplemental media (preferred .png or .gif, <1MB).
2) Media (images, gifs) should accurately represent the existing code blocks in the example sections and fit in-line. ex: [See last image here](https://www.codecademy.com/resources/docs/html/attributes/title)

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [x] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by Jamesllllllllll**
### Description

This is a new PR for the correct file. The [previous PR](https://github.com/Codecademy/docs/pull/4433) was incorrectly editing the main `overflow` term page located at `/content/css/concepts/overflow/overflow.md` - but this one is editing the correct file at `/content/css/concepts/overflow/overflow/overflow.md`

### Issue Solved

Closes #4415 

### Type of Change

- Editing an existing entry (add media)

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

**Comment by Jamesllllllllll** [comment]
Hey @helfese and @avdhoottt - I closed my [previous PR](https://github.com/Codecademy/docs/pull/4433) because it was editing the wrong file. This new PR is editing the correct file!

**Review by ishg-153 — APPROVED** [review]
@Jamesllllllllll good for second review!

File: content/css/concepts/overflow/terms/overflow/overflow.md [hunk 1]
```diff
@@ -50,10 +50,14 @@ Provide a scrollbar and clip any content that overflows.
...
 ```css
 .view-box {
-  overflow: auto;
+  overflow: hidden;
 }
 ```
+
+The following image demonstrates the three `overflow` properties described above:
+
+![Three div elements with overflow properties of visible, scroll and hidden](https://raw.githubusercontent.com/Codecademy/docs/main/media/css-overflow.png)
```
Comment (avdhoottt) [inline content/css/concepts/overflow/terms/overflow/overflow.md:63]:
```suggestion
![Three div elements with overflow properties of visible, scroll, and hidden](https://raw.githubusercontent.com/Codecademy/docs/main/media/css-overflow.png)
```

File: content/css/concepts/overflow/terms/overflow/overflow.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ CatalogContent:
   - 'paths/full-stack-engineer-career-path'
 ---
 
-Defines how a block level element should handle content that goes beyond its boundaries.
+Defines how a block level element should handle content that goes beyond its boundaries. This is shorthand for [`overflow-x`](https://www.codecademy.com/resources/docs/css/overflow/overflow-x) and [`overflow-y`](https://www.codecademy.com/resources/docs/css/overflow/overflow-y).
```
Comment (avdhoottt) [inline content/css/concepts/overflow/terms/overflow/overflow.md:15]:
```suggestion
Defines how a block-level element should handle content that goes beyond its boundaries. This is shorthand for [`overflow-x`](https://www.codecademy.com/resources/docs/css/overflow/overflow-x) and [`overflow-y`](https://www.codecademy.com/resources/docs/css/overflow/overflow-y).
```

**Review by avdhoottt — APPROVED** [review]
LGTM!! Excellent image.

**Pull request merged**